### DISCUSSION TITLE
How to handle FEM mesh in torch_geometric?

### DISCUSSION THREAD

**Discussion by BraveDrXuTF** [pyg-team/pytorch_geometric#9516] [General]
For FEMs, we have all kinds of meshes with different shapes of faces (in 2D) and volumes (in 3D), which are called "elements" in FEMs. For example, we may encounter elements with 3 vertices, 4 vertices, or more in FEM meshes. I want to know if there is an approach in the torch_geometric package to handle FEM meshes. If not, what do you think is necessary to do if I want to work on torch_geometric to extend its functionality to FEMs?

**Comment by rusty1s** [comment]
Do you mean that across examples, faces are either represented using 3 or 4 vertices? PyG wouldn't be able to handle this right now :(
